### DISCUSSION TITLE
GitHub Enterprise Cloud trial - Advanced Security

### DISCUSSION THREAD

**Discussion by rmjuarez12** [community/community#31655] [Enterprise] [answered]
Hello, 

I have a quick question. I have a couple of private repos that I Want to add CodeQL for code analysis. From what I read, the only way I can get it is either by making it public, or by getting Enterprise Server or Cloud. I got the Enterprise Cloud trial, so I could test it out, but it seems that I cannot enable Advanced Security? Is that feature only available to users that buy a membership, or does the trial covers it? I been looking around and, for the life of me, I cannot find the setting!

**Comment by aeisenberg** [comment]
Apologies for the slow response. Thanks for your question. I've passed it on to our field team who should be able to answer this.

**Comment by NickLiffen** [comment, marked as answer]
Thanks for the question @rmjuarez12 👋 

Could you click [Talk to a security expert](https://resources.github.com/security/tools/ghas-trial/) on that page please 🙇 

That should kick off the process 👍
